Makefile

```diff
@@ -838,6 +838,7 @@ LIB_OBJS += base85.o
 LIB_OBJS += bisect.o
```

Comment:
commit-message notes:

typo: therin ==> therein

should we say "changed-path" rather than "changed path" in the open paragraph? 

This commit is all about bloom filter specifically, so perhaps the generic notes in the previous commit aren't as important.